src/lib.rs

```diff
@@ -2,29 +2,15 @@
 //! and Named Pipes for Windows.
 
 #![warn(missing_docs)]
-
-extern crate futures;
-extern crate tokio_uds;
-extern crate tokio_named_pipes;
-extern crate tokio;
-
-extern crate bytes;
-#[allow(unused_imports)] #[macro_use] extern crate log;
-
-#[cfg(windows)]
-extern crate miow;
-#[cfg(windows)]
-extern crate mio_named_pipes;
-#[cfg(windows)]
-extern crate winapi;
+#![deny(rust_2018_idioms)]
```

Comment:
What's that about?

Reply:
The lint name is unfortunate - its meaning is closer to `#![deny(rust_2018_unidiomatic_code)]`.

It warns against, among others, unnecessary `extern crate`s or structs with hidden lifetimes like [here](https://github.com/NikVolf/parity-tokio-ipc/pull/13/files#diff-9fccaa80cc506583c66967ac942a320fR140) (`AceWithSid<'_>`), I used it when transitioning to 2018 edition.

Should I change it to `warn` instead?

Reply:
I'm fine with that, was just curious. It's @NikVolf's call.